jib-maven-plugin/src/test/resources/maven/projects/multi/complex-service/pom.xml

```diff
@@ -22,7 +22,6 @@
             <paths>
               <path>src/main/jib1</path> <!-- should resolve to this projectRoot -->
               <path>${project.basedir}/src/main/jib2</path> <!-- provided as absolute within project -->
-              <path>/some/random/absolute/path/jib3</path> <!-- provided as absolute outside of project -->
```

Comment:
can't use absolute path in the test anymore, because it won't exist in the build server